src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -26,25 +26,29 @@ Connect Sentry to your AWS account to automatically add Sentry error and perform
 
    ![](project.png)
 
-1. Click on **Add Sentry's CloudFormation to your AWS**.
+1. Click on **Go To AWS**.
 
-   ![](aws_account.png)
+   ![](aws_account_1.png)
 
 1. Scroll to the bottom of the Create Stack page, check the box, then press **Create Stack** to create a CloudFormation stack in your current region. This stack allows Sentry to automatically instrument your Lambda functions.
 
    ![](cloudformation.png)
 
-1. Wait for the stack to be created, which is confirmed by seeing the status change to **Create Complete**, then copy your AWS account number, select the region you want to use, and press **Next**.
+1. Wait for the stack to be created, which is confirmed by seeing the status change to **Create Complete**, then click **I've created the stack**.
+
+   ![](aws_account_2.png)
+
+1. Now copy your AWS account number, select the region you want to use, and press **Next**.
 
    ![](stack_created.png)
 
 <Alert title="Note" level="info">
 
-You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, verify that the externalId in the form matches the one for the existing CloudFormation stack on the "Parameters" tab.
+You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, copy the externalId from the existing CloudFormation stack on the "Parameters" tab into the form shown above.
 
 </Alert>
```

Comment:
It's either an Alert or a Note (our old format combined them both into a muddle). From our Tips for Contributors:
## Warning, Note, Important

- **Prerequisites:** use these to list things that are required to finish. Place them prior to the procedure/process that follows so a user doesn't start and stop.
- **Note:** use these for useful context, information that should be visually called out from the core content. Use sparingly.
- **Important:** use these for information that's critical to know (this version introduces breaking changes). Use infrequently.
- **Warning:** use these for items that MUST be well understood before proceeding. These should be used very rarely.

If you determine it's a Note, use the "Note /Note" format (I can't put the full markdown here as GH is interpreting it, but enclose them with the arrow brackets)

Reply:
@PeloWriter so you mean use thee `Note` component instead of the `Alert` component?